profiles/razaahmad333.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+  "issueId": 1285,
+  "name": "Ahmad Raza",
+  "username": "razaahmad333"   
```

Comment:
Trailing whitespace.
```suggestion
  "username": "razaahmad333"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable